crates/client-api/src/routes/subscribe.rs

```diff
@@ -195,6 +195,10 @@ async fn ws_client_actor_inner(
     // awaited, which can lead to bugs like:
     // https://rust-lang.github.io/wg-async/vision/submitted_stories/status_quo/aws_engineer/solving_a_deadlock.html
     //
+    // NOTE: never let this go unpolled while you're awaiting something; otherwise, it's possible
+    //       to deadlock or delay for a long time. see usage of `also_poll()` the branches of the
+    //       `select!` for examples of how to do this.
+    //
```

Comment:
```suggestion
    // NOTE: never let this go unpolled while you're awaiting something; otherwise, it's possible
    //       to deadlock or delay for a long time. see usage of `also_poll()` in the branches of the
    //       `select!` for examples of how to do this.
    //
```